frontend/catalyst/jax_extras/lowering.py

```diff
@@ -108,6 +108,8 @@ def custom_lower_jaxpr_to_module(
     Copyright 2021 The JAX Authors.
     """
 
+    from catalyst.jax_primitives import CUSTOM_LOWERING_RULES
```

Comment:
Circular import?
```suggestion
    # pylint: disable=import-outside-toplevel
    from catalyst.jax_primitives import CUSTOM_LOWERING_RULES
```
or you can try access via `import catalyst` and `catalyst.jax_primitives.CUSTOM_LOWERING_RULES`

Reply:
Yes. Let me give it a try.

Reply:
Worked! Thanks!